Backpack/src/main/java/net/skyscanner/backpack/rating/internal/RatingSelectors.kt

```diff
@@ -72,9 +72,13 @@ internal class RatingSelectors(
   val contentColor: (BpkRating.Score) -> ColorStateList
 
   init {
-    var colorLow = context.getColor(R.color.bpkPanjin)
-    var colorMedium = context.getColor(R.color.__rating_medium_color)
-    var colorHigh = context.getColor(R.color.bpkMonteverde)
+    var colorLow = context.getColor(R.color.bpkHillier)
```

Comment:
all the colours have changed and are now the same in dark + night mode. also added support for theming of the text colours, as these changed now as well and the dark colour doesn't work with the london theme